webapp/pages/terms-and-conditions.vue

```diff
@@ -2,6 +2,7 @@
   <div>
     <ds-space>
       <ds-heading tag="h2">{{ $t('site.termsAc') }}</ds-heading>
+      <small>Version: {{ termsAndConditionsVersion }}</small>
     </ds-space>
     <ds-container>
       <ds-space margin-top="large">
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-07-19T13:32:10Z" title="Friday, July 19th 2019, 3:32:10 pm +02:00">Jul 19, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -1,14 +1,72 @@
 <template>
   <div>
     <ds-space>
-      <ds-heading tag="h2">{{ $t('site.termsAc') }}</ds-heading>
+      <ds-heading tag="h2">{{ $t('site.termsAc') }} ( vom: {{ latestDate_TaC }})</ds-heading>
     </ds-space>
+    <div v-if="confirmed === true">
```


```suggestion
    <div v-if="confirmed">
```